locales/locales.php

```diff
@@ -902,6 +902,18 @@ public function __construct() {
 		$es_mx->google_code = 'es';
 		$es_mx->facebook_locale = 'es_MX';
 
+		$es_pa = new GP_Locale();
+		$es_pa->english_name = 'Spanish (Panama)';
+		$es_pa->native_name = 'Español de Panamá';
+		$es_pa->lang_code_iso_639_1 = 'es';
+		$es_pa->lang_code_iso_639_2 = 'spa';
+		$es_pa->lang_code_iso_639_3 = 'spa';
+		$es_pa->country_code = 'pa';
+		$es_pa->wp_locale = 'es_PA';
```

Comment:
```suggestion
```
This should only be set for locales that are officially supported on translate.wordpress.org.